modules/enterprise/server/jar/src/main/java/org/rhq/enterprise/server/resource/ResourceManagerBean.java

```diff
@@ -764,6 +766,10 @@ private boolean uninventoryResourceBulkDeleteAsyncWork(Subject overlord, int res
 
         boolean hasErrors = false;
         boolean debugEnabled = LOG.isDebugEnabled();
+
+        // Get all packages installed in this resource
+        List<Integer> installedPackages = getInstalledPackages(resourceId);
+
```

Comment:
Because this is a list of packageIds I think it would be more clear if the variable name were 'installedPackageIds'